README.md

```diff
@@ -1,39 +1,90 @@
 # Numeric Separators
 
-This is a [stage 1 proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
+## Stage 1
+
+This is a [proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
```

Comment:
I'd keep the [stage 1 proposal] bit that was added earlier. Seems constructive to give readers a sense of where things are.